.github/workflows/workflow.yml

```diff
@@ -0,0 +1,71 @@
+name: Node CI
+on: [push]
+jobs:
+  ubuntu-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+    steps:
+    - uses: actions/checkout@master
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@master
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Groonga
+      run: |
+        curl --silent --location https://github.com/groonga/groonga/raw/master/data/travis/setup.sh | sh
+        sudo apt-get install -qq -y groonga-normalizer-mysql groonga-tokenizer-mecab
+    - name: npm install and test
+      run: |
+        npm install
+        npm test
+
+  windows-test:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        node-version: [8.x, 10.x, 12.x]
+    steps:
+    - uses: actions/checkout@master
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@master
+      with:
+        node-version: ${{ matrix.node-version }}
+    - name: Install Groonga
+      run: |
+        set ARCHIVE=groonga-9.0.3-65dc3cd-x64.zip
+
+        choco install -y curl 7zip.commandline
+        curl -O https://packages.groonga.org/tmp/%ARCHIVE%
```

Comment:
Released version at https://github.com/groonga/groonga/releases will be better.
e.g. https://github.com/groonga/groonga/releases/download/v9.0.8/groonga-9.0.8-x64.zip built by MinGW or
https://github.com/groonga/groonga/releases/download/v9.0.8/groonga-9.0.8-x64-vs2017.zip built by Visual Studio.

Reply:
I got it.